evalml/tests/conftest.py

```diff
@@ -2226,6 +2241,102 @@ def imputer_test_data():
     return X
 
 
+@pytest.fixture
+def nullable_type_test_data():
+    def _build_nullable_type_data(has_nans=True):
```

Comment:
Yep! The fixture is `nullable_type_test_data` and instead of returning a ready to use DataFrame or Series, it just returns the function `_build_nullable_type_data`, which is what can take in the param. I haven't had to use `has_nans` in this or the other fixture just yet (its intended purpose is to make it easy to pass nullable data types into components that can't handle nans in the future), but in `nullable_type_target`, I use the `ltype` param quite a lot. 

I took the idea for this from `make_data_type` which follows the same pattern.